addon/utils/transition-duration.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+const { testing: isTesting } = Ember;
+
+export default function transitionDuration(duration) {
+  return isTesting ? duration : duration;
```

Comment:
Is this intended to return `0` when `isTesting` is true? It looks like this is returning the `duration` argument either way...

Reply:
Oh no! 😬
I played around with this, and seems I pushed the wrong version 🤦‍♂️
Nice catch, thank you!